test/package.json

```diff
@@ -6,12 +6,13 @@
     "test:cjs": "node index.cjs",
     "test:mjs": "node index.mjs",
     "test:ts": "ts-node --project fixture/tsconfig.node.json index.ts",
-    "test:ts-esm": "ts-node --esm --project fixture/tsconfig.node.json index.ts",
+    "test:ts-esm": "npx tsx --tsconfig fixture/tsconfig.node.json index.ts",
```

Comment:
From the documentation:

> `tsx` is similar to `ts-node --esm --swc`

Reply:
Your changes, makes sense to me @RobinTail. 

Two comments/questions before merging:
1. Can we replace in full `ts-node` to use `tsx`? Just to avoid having 2 dependencies for the same task.
2. Can you move these tools from the `dependencies` to `dev-dependencies` block as they are not needed for distribution?

Reply:
done

Reply:
by the way, the `test` folder is not needed in the distribution completely